doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -538,12 +594,12 @@ gls{component}
 \labelwidthstring 00.00.0000
 \begin_inset CommandInset label
 LatexCommand label
-name "/OF120/"
+name "/OF180/"
 
 \end_inset
 
  If requested by the user, Beagle shuts down the computer it's running on
- after it finished.
+ after it finished a measurement.
```

Comment:
I understand the problem of @flyfl, but I don't think it is very bad. I don't have a better wording.
Maybe following is better: "After finishing the measurements, Beagle is able to shut down the computer it used, if requested by the user."
But I am afraid it is just the same problem in other words.